CoffeeChat/Cells/OnboardingTableViewCell.swift

```diff
@@ -63,23 +68,47 @@ class InterestsTableViewCell: UITableViewCell {
 
         titleLabel.snp.makeConstraints { make in
             make.leading.equalTo(interestImageView.snp.trailing).offset(textSidePadding)
-            make.top.equalTo(cellBackgroundView).inset(sidePadding)
+            if showingInterests {
+                make.top.equalTo(cellBackgroundView).inset(sidePadding)
+            } else {
+                make.centerY.equalToSuperview()
+            }
         }
 
-        categoriesLabel.snp.makeConstraints { make in
-            make.leading.equalTo(titleLabel)
-            make.top.equalTo(titleLabel.snp.bottom)
+        if showingInterests {
+            categoriesLabel.snp.makeConstraints { make in
+                make.leading.equalTo(titleLabel)
+                make.top.equalTo(titleLabel.snp.bottom)
+            }
         }
     }
 
     func configure(with interest: Interest) {
         titleLabel.text = interest.name
         categoriesLabel.text = interest.categories
+        // Determine if a relayout is needed (was showing Group and configured with an Interest)
+        if !initialized || !showingInterests {
+            showingInterests = true
+            initialized = true
+            cellBackgroundView.subviews.forEach { $0.removeConstraints($0.constraints)}
+            setupConstraints(showingInterests: true)
+        }
```

Comment:
Can we make a helper function for this? Since we use the same code for configuring interest and group.